types/types.go

```diff
@@ -457,6 +466,10 @@ func GetShareManagerPodNameFromShareManagerName(smName string) string {
 	return shareManagerPrefix + smName
 }
 
+func GetConfigMapNameFromShareManagerName(smName string) string {
+	return shareManagerPrefix + smName
```

Comment:
Because you actually use the share manager name, I would suggest  changing the code as below to make it explicitly.

```golang
return GetShareManagerPodNameFromShareManagerName(smName)
```

Reply:
No prob. Maybe add a prefix, e.g. "recovery-backend-<share manager name>". How about it?

Reply:
Sounds good to know if the length is allowed.

Reply:
Updated. Thank you.